esp-hal/CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Breaking
```

Comment:
This should just be under `## Removed`, let's not invent new headings to put in the changelogs please :)

Reply:
This heading is already in `CHANGELOG`, but I agree, we haven't used it in a while, so let's move this under “Removed” 😄

Reply:
Don't know who put it in there, but these are [all standardized](https://keepachangelog.com/en/1.0.0/) and we should follow that IMO. Thanks for updating that.